sensor_msgs/README.md

```diff
@@ -0,0 +1,51 @@
+
+## sensor_msgs c++ API
+This package provides some common c++ functionality relating to manipulating a couple of particular sensor_msgs messages.
+
+* [fill_image.hpp](include/sensors_msgs/fill_image.hpp): Fill a Image message from type-erased data pointer.
+* [image_encodings.hpp](include/sensor_msgs/image_encodings): Definitions and functionality relating to image encodings.
+* [point_cloud_conversion.hpp](include/sensor_msgs/point_cloud_conversion.hpp): Functionality for converting between the deprecated PointCloud and PointCloud2 messages.
+* [point_cloud2_iterator.hpp](include/sensor_msgs/point_cloud2_iterator.hpp): Tools for modifying and parsing PointCloud2 messages.
+* [point_field_conversion.hpp](include/sensor_msgs/point_field_conversion.hpp): A type to enum mapping for the different PointField types, and methods to read and write in a PointCloud2 buffer for the different PointField types.
+
+## Messages (.msg)
+* [BatteryState](BatteryState.msg): Describes the power state of the battery.
+* [CameraInfo](CameraInfo.msg): Meta information for a camera.
+* [ChannelFloat32](ChannelFloat32.msg): Holds optional data associated with each point in a PointCloud message.
+* [CompressedImage](CompressedImage.msg): A compressed image.
+* [FluidPressure](FluidPressure.msg): Single pressure reading for fluids (air, water, etc) like atmospheric and barometric pressures.
+* [Illuminance](Illuminance.msg): Single photometric illuminance measurement.
+* [Image](Image.msg): An uncompressed image.
+* [Imu](Imu.msg): Holds data from an IMU (Inertial Measurement Unit).
+* [JointState](JointState.msg): Holds data to describe the state of a set of torque controlled joints.
+* [JoyFeedbackArray](JoyFeedbackArray.msg): An array of JoyFeedback messages.
+* [JoyFeedback](JoyFeedback.msg): Describes user feedback in a joystick, like an LED, rumble pad, or buzzer.
+* [Joy](Joy.msg): Reports the state of a joystick's axes and buttons.
+* [LaserEcho](LaserEcho.msg): A submessage of MultiEchoLaserScan and is not intended to be used separately.
+* [LaserScan](LaserScan.msg): Single scan from a planar laser range-finder.
+* [MagneticField](MagneticField.msg): Measurement of the Magnetic Field vector at a specific location.
+* [MultiDOFJointState](MultiDOFJointState.msg): Representation of state for joints with multiple degrees of freedom, following the structure of JointState.
+* [MultiEchoLaserScan](MultiEchoLaserScan.msg): Single scan from a multi-echo planar laser range-finder.
+* [NavSatFix](NavSatFix.msg): Navigation Satellite fix for any Global Navigation Satellite System.
+* [NavSatStatus](NavSatStatus.msg): Navigation Satellite fix status for any Global Navigation Satellite System.
+* [PointCloud2](PointCloud2.msg): Holds a collection of N-dimensional points, which may contain additional information such as normals, intensity, etc.
+* [PointCloud](PointCloud.msg): ** ## THIS MESSAGE IS DEPRECATED AS OF FOXY, use PointCloud2 instead**
```

Comment:
Use the same deprecation message here and above

Reply:
Done